var/spack/repos/builtin/packages/intel-mkl/package.py

```diff
@@ -83,6 +83,34 @@ def blas_libs(self):
     def lapack_libs(self):
         return self.blas_libs
 
+    @property
+    def scalapack_libs(self):
+        libnames = [ 'libmkl_scalapack' ]
+        if self.spec.satisfies('^openmpi'):
+            libnames.append('libmkl_blacs_openmpi')
+        elif self.spec.satisfies('^mpich@1'):
+            libnames.append('libmkl_blacs')
+        elif self.spec.satisfies('^mpich@2:'):
+            libnames.append('libmkl_blacs_intelmpi')
```

Comment:
i recall that i was compiling `trilinos` with `mkl_scalapack_lp64;mkl_blacs_mpich_lp64`, so i would expect `mkl_blacs_mpich` to appear somewhere in this list.